app/javascript/packs/document-capture.tsx

```diff
@@ -188,6 +189,13 @@ const App = composeComponents(
     {
       value: {
         exitQuestionSectionEnabled: String(uiExitQuestionSectionEnabled) === 'true',
+      },
+    },
+  ],
+  [
+    SelfieCaptureEnabledContext.Provider,
+    {
+      value: {
         selfieCaptureEnabled: getSelfieCaptureEnabled(),
```

Comment:
Would it benefit us to make this slightly more generic to any values relating to selfie capture, not just whether it's enabled? So we could add other context relating to selfie capture, if needed. (And save a few keystrokes when typing the name 😄 )

```suggestion
    SelfieCaptureContext.Provider,
    {
      value: {
        selfieCaptureEnabled: getSelfieCaptureEnabled(),
```

I'd also wonder if it being within the context makes the "selfieCapture" prefix of `selfieCaptureEnabled` redundant. I think that's true, but I also think in practical usage it's useful to have that qualification when using it in a component with a bunch of other variables. What do you think?

e.g.

```tsx
<SelfieCaptureEnabledContext.Provider value={{ isEnabled: ... }} />

const { isEnabled } = useContext(SelfieCaptureEnabledContext);
// vs. const { isSelfieCaptureEnabled } = useContext(SelfieCaptureEnabledContext);
// vs. const { isEnabled: isSelfieCaptureEnabled } = useContext(SelfieCaptureEnabledContext);
```

Reply:
@aduth Those are great points! I'm going to push up a change for the first one.

Your second point is great too, I'm just less sure of the answer. Is one of the options more conventional in React? I think using just `isEnabled` would be clear in the context file itself, but less clear where it's used, as you mention, with a lot of other variables.

So my inclination is to use one of your second two options. But I'm not sure which is more "React"-like. I actually didn't know you could rename it like in the third option, so that's pretty neat!

Reply:
I don't know that there's a React convention related to this, it's probably more a judgment call on our part based on our own usage. Personally I don't feel too strongly one way or the other.

We've got at least one [example](https://github.com/18F/identity-idp/blob/4a1ba4a33a789ad4c1fda71a31ba06f7978d5c47/app/javascript/packages/document-capture/components/document-capture-abandon.tsx#L30) of the rename approach, but there's some [examples](https://github.com/18F/identity-idp/blob/4a1ba4a33a789ad4c1fda71a31ba06f7978d5c47/app/javascript/packs/document-capture.tsx#L125-L127) of others which "redundant" qualification.

If forced to pick, I'd say it's probably common enough for these context values to be used within components that aren't strictly related to selfie capture where having the qualification would be useful to disambiguate amongst variables in the same scope.

Reply:
@night-jellyfish , I think `isEnabled` may be ok, since when you use it in other component, you can assign it to a variable whose name can be anything intended.

Reply:
Thank you for sharing your input!

These changes were made in:

1. [de30c24](https://github.com/18F/identity-idp/pull/10183/commits/de30c24a7d9b781615048d752a652c186e8613a9)
2. [e98f9af](https://github.com/18F/identity-idp/pull/10183/commits/e98f9af432107b7446074b826b1e5cf4bf7d6516)

(with some fixup commits after).